DecipherNow/gm-fabric-dashboard
# Update Publish Script to add more robust publish logic

Resolves #983 

## Proposed Changes

  - Replace existing basic BASH script with a Node.js script that allows for dynamic SemVer tagging on DockerHub
  - Add dryRun functionality to give users greater confidence about the arguments
  
## How to Test

  - Review the options from lines 132-146 of publish.sh
  - Experiment with different combinations using --dryRun
  - Once comfortable, run without --dryRun, but don't run with --confirm and answer no to the confirmation prompt.
